alf/trainers/policy_trainer.py

```diff
@@ -718,8 +719,8 @@ def play(root_dir,
         logging.info(
             "%s: %s", m.name,
             map_structure(
-                lambda x: x.numpy().item() if x.ndim == 0 else x.numpy(),
-                m.result()))
+                lambda x: x.cpu().numpy().item()
+                if x.ndim == 0 else x.cpu().numpy(), m.result()))
```

Comment:
It's strange that how the previous code can work.

Reply:
> It's strange that how the previous code can work.

If play.py is called in CPU mode, then it won't be any problem